modules/otel-configuring-oteloperator-metrics.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * otel/otel-configuring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-oteloperator-metrics_{context}"]
+= Configuring the {OTELOperator} metrics and alerts
+
+When installing the {OTELOperator} from the web console, you can select the *Enable Operator recommended cluster monitoring on this Namespace* checkbox, which enables creating metrics of the {OTELOperator}.
```

Comment:
The checkbox to enable operator recommended monitoring is not present on OCP web console for OpenTelemetry operator. https://issues.redhat.com/browse/TRACING-3761 While this issue is resolved, we can modify the steps to set the monitoring label on the namespace in which the operator is installed, by default openshift-opentelemetry-operator

Reply:
@IshwarKanse, I'm removing this. Is the following still correct? 
```
= Configuring the {OTELOperator} metrics and alerts

You can manually enable metrics and alerts after installing the {OTELOperator}.

.Procedure

* Add the `openshift.io/cluster-monitoring: "true"` label in the project where the {OTELOperator} is installed, which is `openshift-opentelemetry-operator` by default.

.Verification

You can use the *Administrator* view of the web console to verify successful configuration:

. Go to *Observe* -> *Targets*, filter by *Source: Platform*, and search for `opentelemetry-operator`, which must have the *Up* status.
```

Reply:
It doesn't work with the label. We are checking on a workaround for this.